pkg/providers/aws/plugin.go

```diff
@@ -25,26 +25,13 @@ import (
 	sqs_service "github.com/nitrictech/nitric/pkg/plugins/queue/sqs"
 	"github.com/nitrictech/nitric/pkg/plugins/storage"
 	s3_service "github.com/nitrictech/nitric/pkg/plugins/storage/s3"
-	"github.com/nitrictech/nitric/pkg/providers"
 	"github.com/nitrictech/nitric/pkg/providers/aws/core"
 )
 
 type AWSServiceFactory struct {
 	provider core.AwsProvider
 }
 
-func New() (providers.ServiceFactory, error) {
-	provider, err := core.New()
-
-	if err != nil {
-		return nil, err
-	}
-
-	return &AWSServiceFactory{
-		provider: provider,
-	}, nil
-}
-
```

Comment:
If the code in these is dead, we can probably remove these.

Initially wanted to look at using something like `wire` to do compile time dependency injection for building the different provider membranes, but we can clean out this dead code for now.